src/main/java/com/dabsquared/gitlabjenkins/connection/GitLabConnection.java

```diff
@@ -316,14 +268,5 @@ public ListBoxModel doFillApiTokenIdItems(@QueryParameter String url, @QueryPara
             }
             return new StandardListBoxModel();
         }
-
-        public ListBoxModel doFillClientBuilderIdItems() {
-            ListBoxModel model = new ListBoxModel();
-            for (GitLabClientBuilder builder : getAllGitLabClientBuilders()) {
-                model.add(builder.id());
-            }
-
-            return model;
-        }
```

Comment:
As described above, we cannot break compatibility nor is it desirable to drop support for autodetection. This should be reverted.